source/default/_patterns/00-protons/index.js

```diff
@@ -1,9 +1,32 @@
 /**
- * Base css generation and global js logic
+ * Base css generation and global js logic.
  */
 
 import './_base.scss';
 
+import { mediaBreakpoint } from 'protons/utilities';
+import scssVariables from '../../_data/scssVariables.json';
+
+import enquire from '../../../node_modules/enquire.js';
+
+export function enable() {
```

Comment:
The "protons" concept doesn't execute an `enable()` function, as it shouldn't need to wait for document.ready(). Just run this logic at root of `protons/index.js`

Reply:
@illepic, updated this. Note that it appears running this in proton makes the first "match" not hit until the page gets fully loaded, but at least we have an example where the functionality is defined.